server/controllers/api/assessmentDayDataController/assessmentDayDataController.test.js

```diff
@@ -385,19 +385,27 @@ describe('assessmentDayDataController', () => {
 
         const assessmentVariables = await appDb
           .collection(collections.assessmentVariables)
-          .find({ assessment: newAssessment }, { projection: { _id: 0 } })
+          .find(
+            { assessment_id: assessmentDoc._id },
+            { projection: { _id: 0, assessment_id: 0 } }
+          )
           .toArray()
 
-        expect(assessmentVariables).toEqual(
-          newAssessmentVariables.map(({ name, assessment }) => ({
-            name,
-            assessment,
-          }))
+        expect(assessmentVariables.sort()).toEqual(
+          newAssessmentVariables.sort()
         )
       })
       it("will not make any updates if the assessment variables from the importer are the same as what's in the database", async () => {
         const noActionAssessment = 'no-action'
-        const variables = initialAssessmentVariables(noActionAssessment)
+        const assessmentAttributes = AssessmentModel.withDefaults({
+          name: noActionAssessment,
+        })
+        const assessmentDoc = await AssessmentModel.upsert(
```

Comment:
again this seems to be testing the model's upsert method and not the controller action.

Reply:
Okay, for this test my goal is to set up the documents in the database and I used the model to insert the test data. 

What do you suggest I do to test this..?

Reply:
https://github.com/AMP-SCZ/dpdash/pull/732#discussion_r1566282386